doc/MANUAL.adoc

```diff
@@ -881,6 +934,34 @@ Examples:
 *file_stat_matches_ctime*::
     Ignore ctimes when *file_stat_matches* is enabled. This can be useful when
     backdating files' mtimes in a controlled way.
+[#config_sloppy_incbin]
+*incbin*::
+    Usually, in <<the_preprocessor_mode,preprocessor mode>> and
+    <<the_unify_mode,unify mode>>, ccache scans the preprocessed file
+    for assambler
+    https://sourceware.org/binutils/docs/as/Incbin.html[*.incbin*]
+    directives (e.g. in *asm* inline assembly statements). The effect of
+    the directive is to emit the referenced file without any processing
+    into the object file and is sometimes used to more efficiently
+    include a binary file more efficiently than by storing it in a C/C++
+    constant array. However, the inclusion is not done by the
+    preprocessor and therefore its content not inserted into the
+    preprocessed output nor in the dependency information inserted into
+    the *.d* generated by *-MD*/*-MMD* (Used in
+    <<the_unify_mode,unify mode>>, <<the_direct_mode,*direct_mode*>> and
+    nearly every buildsystem).
++
+The incbin sloppiness option allows ccache to skip scanning for
+the .incbin directive. This is mostly beneficial in
+<<the_unify_mode,unify mode>> where searching for *.incbin* is the only
+reason for it to parse the preprocessed file and therefore could be skipped.
++
+Neither <<_the_direct_mode,direct mode>> nor
+<<the_depend_mode,depend mode>> take *.incbin*
```

Comment:
If the file gets into the cache via depend mode, the manifest will not include the `.incbin` file and therefore get a direct hit in direct mode even though the `.incbin` file may have changed. Therefore I think wording is correct, direct mode does not look for `.incbin` instruction, it just hashes the files in the manifest.

This might be an issue with calling them "modes".